src/node/mini_miner.h

```diff
@@ -72,7 +75,10 @@ class MiniMiner
     // the same tx will have the same bumpfee. Excludes non-mempool transactions.
     std::map<uint256, std::vector<COutPoint>> m_requested_outpoints_by_txid;
 
-    // What we're trying to calculate.
+    // Txid to a sequence number representing the order in which this transaction was included.
```

Comment:
I think I might not the first one to make the comment, though using “sequence number” can be confusing as we have already the transaction `nSequence`. I guess order number or ancestor number could be used.

Reply:
Renamed to `m_inclusion_order`